examples/two-step-transfer/index.html

```diff
@@ -64,7 +64,8 @@
                         .then(alert)
                         .catch(alert)
                 );
-                provider.addChangeAccountListener(alert);
+                provider.on('accountChanged', (accountAddres) => alert(accountAddres));
+                provider.on('chainChanged', (chain) => alert(chain));
             }
```

Comment:
:cry:

Reply:
Yes, it is sad that `alert` assumes that `this` refers to `window`.